packages/nimble-components/src/anchor/specs/README.md

```diff
@@ -0,0 +1,211 @@
+# Nimble Anchor
+
+## Overview
+
+The Nimble Anchor is a component used to navigate to a web resource, similar to the HTML anchor element.
+
+### Background
+
+[Visual Design spec](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/bfadf499-caf5-4ca0-9814-e777fbae0d46)<br>
+[Nimble work item](https://github.com/ni/nimble/issues/324)
+
+---
+
+## Design
+
+Initially we will create two flavors of anchor: a standard anchor, and a button-like anchor. These will be two separate components. For justification, see the Future Considerations section at the end of this document.
+
+### API
+
+[FAST's API documentation](https://github.com/microsoft/fast/blob/e576aa70c22780fffba03097277e2db9a2ec1cd8/packages/web-components/fast-foundation/src/anchor/README.md)
+
+-   _Component Name_: `nimble-anchor`
+-   _Properties/Attributes_: We will have the following properties/attributes in addition to the ones provided by the FAST anchor:
+    -   `inline`: specifies that the anchor will be used inline within a text block (as opposed to a standalone element in the UI). The anchor will always have an underline and will be formatted to align with surrounding text.
+    -   `prominent`: the "loud" version from the design doc. Colors the link text green. The name of this attribute comes from the Breadcrumb's `appearance` attribute value.
```

Comment:
I was kind of torn between what I have now and what you suggest. Since we only have one value, it seemed simpler just to make the attribute itself `prominent`, but consistency is also good. I'll change it.